### PR TITLE
Updating quick starts for latest okta-spring-boot-starter release

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

File: _source/quickstart/java/spring-auth-code.md [hunk 1]
```diff
@@ -3,48 +3,33 @@ layout: quickstart_partial
...
-      # from your Auth Server's metadata, check .well-known/openid-configuration 
-      # if not in .well-known/oauth-authorization-server
-      userInfoUri: # userinfo_endpoint
-      preferTokenInfo: false
+```properties
+okta.oauth2.issuer=https://{yourOktaDomain}.com/oauth2/default
+okta.oauth2.clientId={yourClientId}
+okta.oauth2.clientSecret={yourClientSecret}
+# Configure the callback URL to match the previous section
+security.oauth2.sso.loginPath=/authorization-code/callback
```
Comment (jmelberg-okta) [inline _source/quickstart/java/spring-auth-code.md]: Figuring this is a Spring Security thing, but why this is named `loginPath`? Doesn't Spring give you the ability to specify the callback route?

File: _config.yml [hunk 2]
```diff
@@ -99,7 +99,7 @@ defaults:
 # Okta Library versions
 versions:
   jwt_validator_java: 0.2.0
-  spring_security_starter: 0.1.0
+  spring_boot_starter: 0.2.0
```
Comment (jmelberg-okta) [inline _config.yml]: Thank you for this! 🎉

File: _source/quickstart/java/spring-implicit.md [hunk 3]
```diff
@@ -25,11 +25,11 @@ You can configure your applications properties with environment variables, syste
...
 | Property | Default | Details |
 |----------|---------|---------|
-| okta.oauth.issuer     | N/A | [Authorization Server](/docs/how-to/set-up-auth-server.html) issuer URL, i.e.: https://{yourOktaDomain}.com/oauth2/default |
-| okta.oauth.clientId   | N/A | The Client Id of your Okta OIDC application |
-| okta.oauth.audience   | api://default | The audience of your [Authorization Server](/docs/how-to/set-up-auth-server.html) |
-| okta.oauth.scopeClaim | scp | The scope claim key in the Access Token's JWT |
-| okta.oauth.rolesClaim | groups | The claim key in the Access Token's JWT that corresponds to an array of the users groups. |
+| okta.oauth2.issuer     | N/A | [Authorization Server](/docs/how-to/set-up-auth-server.html) issuer URL, i.e.: https://{yourOktaDomain}.com/oauth2/default |
+| okta.oauth2.clientId   | N/A | The Client Id of your Okta OIDC application |
+| okta.oauth2.audience   | api://default | The audience of your [Authorization Server](/docs/how-to/set-up-auth-server.html) |
```
Comment (jmelberg-okta) [inline _source/quickstart/java/spring-implicit.md]: Default as in Spring Security `default`, or Okta `default`?

File: _source/quickstart/java/spring-auth-code.md (hunk 1 above)
Comment (bdemers) [reply _source/quickstart/java/spring-auth-code.md]: What happens is when a user is denied access, they are redirected to `/authorization-code/callback` (within the application) that route [the filter for the route] then redirects to the remote OAuth URL, then returns back to `/authorization-code/callback`

File: _source/quickstart/java/spring-implicit.md (hunk 3 above)
Comment (bdemers) [reply _source/quickstart/java/spring-implicit.md]: Yes ;)  the default values of the properties when not explicitly set

File: _source/quickstart/java/spring-auth-code.md (hunk 1 above)
Comment (jmelberg-okta) [reply _source/quickstart/java/spring-auth-code.md]: Interesting. Essentially if someone navigates over to `/authorization-code/callback` without any query params, it'll step into the authorization flow?

Comment (bdemers) [reply _source/quickstart/java/spring-auth-code.md]: Correct, after the actual callback the user will be redirected to `/` by default

**Pull request merged**